data/presets/landuse/meadow_orchard/meadow.json

```
@@ -0,0 +1,37 @@
+{
+    "icon": "maki-park",
+    "fields": [
+        "name",
+        "operator",
+        "trees"
+    ],
+    "moreFields": [
+        "{@templates/contact}",
+        "address",
+        "species/wikidata",
+        "surface",
+        "description"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "tags": {
+        "landuse": "meadow",
+        "meadow": "meadow_orchard"
+    },
+    "reference": {
+        "key": "meadow",
+        "value": "meadow_orchard"
+    },
+    "terms": [
+        "fruit",
+        "maedow"
+    ],
+    "name": "Meadow with fruit trees",
```

preset names should be title case

```suggestion
    "name": "Meadow With Fruit Trees",
```